src/test/ui/issues/issue-41974.rs

```diff
@@ -1,14 +1,14 @@
 #[derive(Copy, Clone)]
 struct Flags;
 
-trait A {
-}
+trait A {}
 
-impl<T> Drop for T where T: A { //~ ERROR E0119
-                                //~^ ERROR E0120
-                                //~| ERROR E0210
-    fn drop(&mut self) {
-    }
+impl<T> Drop for T where T: A { 
+    //~^ ERROR auto traits must not contain where bounds
```

Comment:
this error message is *definitely* confusing =)

Reply:
am I missing something here? It seems like auto traits have nothing to do with this

Reply:
no you are not, not sure why I worded it this way :sweat_smile: 

Will now actually spend some time cleaning this up